stories/RingGraph/RingGraph.mdx

```diff
@@ -0,0 +1,33 @@
+{/* Generated with scripts/create-componnent.js */}
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import  RingGraph  from '../../components/RingGraph';
+
+import { Header } from '../shared';
+
+import { 
+  Default
+} from './RingGraph.stories.jsx';
+
+<Meta title="RingGraph" component={RingGraph}  />
```

Comment:
The position of the Score component, in storybook list, should be in:
Ring Graph (folder)
- Stepper (move Stepper component to inside the folder Ring Graph)
- Score ( this component )